scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -417,7 +416,11 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                                     .split(',');
 
                                 for (var i = 0; i < modifiedErrors.length; i++) {
-                                    notify.error(_.trim(modifiedErrors[i]));
+                                    var message = _.trim(modifiedErrors[i]);
+                                    var field = helpers.getFirstWord(message).substr(1);
+
+                                    $scope.error[field.toLowerCase()] = true;
+                                    notify.error(message);
```

Comment:
What is going on here? Why are we displaying only the first word without the first character? It's not self-explanatory.